test/test_lyrics.py

```diff
@@ -395,6 +423,44 @@ def test_is_page_candidate_special_chars(self):
         google.is_page_candidate(url, url_title, s['title'], u'Sunn O)))')
 
 
+class LyricsGeniusBaseTest(unittest.TestCase):
+
+    def setUp(self):
+        """Set up configuration."""
+        try:
+            __import__('bs4')
+        except ImportError:
+            self.skipTest('Beautiful Soup 4 not available')
+        if sys.version_info[:3] < (2, 7, 3):
+            self.skipTest("Python's built-in HTML parser is not good enough")
+
+
+class LyricsGeniusScrapTest(LyricsGeniusBaseTest):
+
+    """Checks that Genius backend works as intended.
+    """
+    import requests
+
+    def setUp(self):
+        """Set up configuration"""
+        LyricsGeniusBaseTest.setUp(self)
+        self.plugin = lyrics.LyricsPlugin()
+
+    @patch.object(requests, 'get', GeniusMockGet())
+    def test_no_lyrics_div(self):
+        """Ensure that `lyrics_from_song_api_path` doesn't crash when the html
+        for a Genius page contain <div class="lyrics"></div>
+        """
+        # https://github.com/beetbox/beets/issues/3535
+        # expected return value None
+        try:
+            self.assertEqual(genius.lyrics_from_song_api_path('/nolyric'),
+                             None)
+        except AttributeError:
```

Comment:
Instead of catching the exception and assessing, I think we can just let the exception through. That is, you can remove the `try`/`except` handler here completely—the test will error out if the bug exists.

Reply:
Sure thing!